packages/ffe-form-react/src/StepSelector.md

```diff
@@ -0,0 +1,31 @@
+Dette er en numerisk, horisontal velger som ble bygd for å velge antall barn. Passer best til ha tall som verdier, tåler ikke større felt enn tall.
+
+```js
+const { RadioButtonInputGroup } = require('.');
+initialState = { selected: undefined };
+<RadioButtonInputGroup
+    inline={true}
+    name="antallBarn"
+    label="Hvor mange barn har du ?"
+    description="Her kan du velge antall barn du skal ha"
+    onChange={e => setState({ selected: e.target.value })}
+    selectedValue={state.selected}
+>
+    {inputProps => (
+        <div className="ffe-step-selector">
```

Comment:
varfør trenger du denne?
`<div className="ffe-step-selector">`

Jag fikk det til uten med:
```
.aman-radio-button {
    display: inline-flex;
    background: red;
    height: 48px;
    align-items: center;
    padding: 0 24px;
    cursor: pointer;

    &:first-of-type {
        border-radius: 24px 0 0 24px;
    }

    &:last-of-type {
        border-radius: 0 24px 24px 0;
    }
}

.ffe-radio-input:checked + .aman-radio-button {
    background: blue;
    color: white;
}
```

Reply:
Trodde denne trengs for å style optimalt. Men fjernes nå.